aya/src/util.rs

```diff
@@ -95,6 +95,36 @@ fn parse_kernel_symbols(reader: impl BufRead) -> Result<BTreeMap<u64, String>, i
     Ok(syms)
 }
 
+/// Find current system's syscall prefix by testing on the BPF syscall.
```

Comment:
This should return a io::ErrorKind::Other or maybe even io::ErrorKind::NotFound if a prefix can't be found. Also the docs should be something like:

```
/// Returns the prefix used by syscalls.
///
/// # Example
///
/// ...example that shows how to use format to prefix a syscall with this prefix...
///
/// # Errors
///
/// Returns an [`std::io::Error`] if kernel symbols can't be found or the prefix can't be determined.
```